test/e2e/runner/perturb.go

```diff
@@ -59,7 +59,11 @@ func PerturbNode(node *e2e.Node, perturbation e2e.Perturbation) (*rpctypes.Resul
 
 	case e2e.PerturbationRestart:
 		logger.Info(fmt.Sprintf("Restarting node %v...", node.Name))
-		if err := execCompose(testnet.Dir, "restart", node.Name); err != nil {
+		if err := execCompose(testnet.Dir, "kill", "-s", "SIGTERM", node.Name); err != nil {
```

Comment:
Could we achieve this effect without the sleep, using the `restart` subcommand?

Reply:
possibly!

I'm far more interested in being explicit and uniform between options than I am in testing the behavior of various docker compose operations. I think just using SIGTERM captures the real spirit of the dimension that we're trying 

(for separate reasons, it would be nice to be able to orchestrate the tests without docker compose, so we could run multiple test nets concurrently on one system, given enough resources, and reduce the build time (to promote improved parallelism in CI), so I'm perhaps even less invested in taking advantage of docker features than I might otherwise be.)

Reply:
👍🏼 for maybe ditching docker-compose at some point. A sleep-wait just makes me worry about test brittleness. The fact that it's ten seconds makes that less likely—at the cost of possibly slowing an already-very-slow test. So not that I think `restart` is inherently preferable, just that it doesn't rely on an unknown timeout.

But this is probably fine in practice.